src/db/tables/onerep_scans.ts

```diff
@@ -49,14 +49,14 @@ async function getLatestOnerepScanResults(
   const results =
     typeof scan === "undefined"
       ? []
-      : await knex("onerep_scan_results")
-          .select()
+      : ((await knex("onerep_scan_results")
+          .select("onerep_scan_results.*")
```

Comment:
I think it'd be better to be explicit about the column names that you want to return - from the docs, it looks like you should be able to specify multiple column names here as an array of `table.column`.

Reply:
@rhelmer This function is used all over the website everywhere we want to display scans - I think we'll want to return all of them, regardless of what columns exist, now or in the future? (We were already doing that before this PR, by the way - and also all columns from the joined tables, except where they had the same column names - then they overwrote eachother.)